app/styles/_bootstrap-overrides.scss

```diff
@@ -0,0 +1,3 @@
+.btn {
+  text-transform: capitalize;
```

Comment:
Curious what is the purpose of this? And if there is an obvious design change to the UI, please post a screencap.